### DISCUSSION TITLE
resource detail metrics do not reload after action finishes

### DISCUSSION THREAD

**Discussion by vesper8** [laravel/nova-issues#3650] [Q&A] [answered]
I have 3 trend metrics on my resource detail card, and I've noticed that if I run an action that affects the data being displayed in the metrics card, once the action finishes the resource data itself is updated but the metrics are not updated and require a manual page reload before they will show the new data.

**Comment by vesper8** [comment]
@crynobone You wrote a comment saying this is already available, but don't explain how to actually make it work?

I'm using the very latest version of Nova 3.30 and it is not happening. The metric cards do not get refreshed following a custom action. A manual page reload is required. So what am I missing?

Thank you

**Comment by crynobone** [comment, marked as answer]
Documentation PR: https://github.com/laravel/nova-docs/pull/351

Example: https://github.com/laravel/nova-dusk-suite/commit/5f3e90eb3db6cd913a0e5fb60d85643aecd1f610

https://user-images.githubusercontent.com/172966/142873206-404ca9c2-9d0f-4f6d-b037-e31bce0f33a7.mp4

**Reply by vesper8** [reply]
thank you!
